api/src/test/java/org/sonarsource/scanner/api/UtilsTest.java

```
@@ -76,13 +74,6 @@ public void task_should_not_require_project() {
     assertThat(Utils.taskRequiresProject(props)).isFalse();
   }
 
-  @Test
-  public void delete_quietly() {
-    Path f = mock(Path.class);
-    doThrow(IOException.class).when(f).getFileSystem();
-    Utils.deleteQuietly(f);
```

Why have you removed this test?

---

I've upgraded all dependencies to be up to date. And now Mockito is checking that the signature of the method is allowing to throw the kind of exception we ask it to throws. 

In this case `getFileSystem()` cannot throw an `IOException`, we would like that `Files#walkFileTree` to throw an `IOException` to check that `deleteQuietly` is resilient to such case.
However we have to use PowerMockito to mock static methods, I though it was a little too much to include PowerMockito to test this. 

Obviously, if you think that this test must remain, I can either add PowerMockito or downgrading Mockito, as you likes.

---

Can't you throw another exception instead? Any runtime exception would fit. Here I think the purpose is to be sure deleteQuietly will silently ignore all exceptions.

---

I've tried that, but `deleteQuietly` is only catching `IOException`, so in this case the test is failing with a RuntimeException not catched.  :(

---

ok, then remove the test, I have no better idea...